ATL/schema.sql

```diff
@@ -0,0 +1,96 @@
+CREATE DATABASE ATL;
+USE ATL;
+
+
+CREATE TABLE Passengers (
+    Id INT IDENTITY(1,1) PRIMARY KEY NOT NULL,
+    FirstName VARCHAR(15),
+    LastName VARCHAR(15),
+    Age INTEGER

```

Comment:
Storing age like this is a really bad idea. Try to think about why that is, we'll discuss it during the class.